src/vaishnaviVJun20_ExceptionHandling/SumOfFirstThreeNumbers.java

```diff
@@ -0,0 +1,44 @@
+package vaishnaviVJun20_ExceptionHandling;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+
+/* WAP to get first 3 numbers from user. Print 3 numbers and their sum.(Use exception handling concept. )
+Note : User may enter wrong inputs, make sure program should not break.
+
+Input :
+10
+TechnoCredits
+2tech3
+30
+40
+Output : values are 10,30,40 and total is 80 */
+
+import java.util.Scanner;
+
+public class SumOfFirstThreeNumbers {
+	Scanner scanner = new Scanner(System.in);
+	int count = 0, sum = 0;
+	int[] array = new int[3];
+
+	void getSumOfFirstThreeNumbers() {
+		System.out.println("Please enter numbers to add: ");
+		do {
```

Comment:
Good u used do while loop
Different than other's

Reply:
Thank you so much @harshvegada :)